app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizardBody.js

```diff
@@ -69,6 +35,7 @@ PlanWizardBody.propTypes = {
   plansBody: PropTypes.object,
   planWizardGeneralStep: PropTypes.object,
   planWizardVMStep: PropTypes.object,
```

Comment:
The redux-form related props were lifted to `PlanWizard`.  Should we remove the prop types and default props?

Reply:
Ah, true! Good call.